source/module_cell/klist.cpp

```diff
@@ -45,64 +46,64 @@ K_Vectors::~K_Vectors()
 #endif
 }
 
-void K_Vectors::set(
-    const ModuleSymmetry::Symmetry &symm,       // symmetry module used for generating k-points.
-    const std::string &k_file_name,             // input file：KPT file
-    const int& nspin_in,
-    const ModuleBase::Matrix3 &reciprocal_vec,
-    const ModuleBase::Matrix3 &latvec)
+void K_Vectors::set(const ModuleSymmetry::Symmetry& symm,
+                    const std::string& k_file_name,
+                    const int& nspin_in,
+                    const ModuleBase::Matrix3& reciprocal_vec,
+                    const ModuleBase::Matrix3& latvec)
 {
     ModuleBase::TITLE("K_Vectors", "set");
 
-	GlobalV::ofs_running << "\n\n\n\n";
-	GlobalV::ofs_running << " >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>" << std::endl;
-	GlobalV::ofs_running << " |                                                                    |" << std::endl;
-	GlobalV::ofs_running << " | Setup K-points                                                     |" << std::endl;
-	GlobalV::ofs_running << " | We setup the k-points according to input parameters.               |" << std::endl;
-	GlobalV::ofs_running << " | The reduced k-points are set according to symmetry operations.     |" << std::endl;
-	GlobalV::ofs_running << " | We treat the spin as another set of k-points.                      |" << std::endl;
-	GlobalV::ofs_running << " |                                                                    |" << std::endl;
-	GlobalV::ofs_running << " <<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<" << std::endl;
-	GlobalV::ofs_running << "\n\n\n\n";
-
-	GlobalV::ofs_running << "\n SETUP K-POINTS" << std::endl;
-
-	// (1) set nspin, read kpoints.
-	this->nspin = nspin_in;
-	ModuleBase::GlobalFunc::OUT(GlobalV::ofs_running,"nspin",nspin);
-	if(this->nspin==4)
-	{
-		this->nspin = 1;//zhengdy-soc
-	}
+    GlobalV::ofs_running << "\n\n\n\n";
```

Comment:
Thanks for your effort, I am doing something similar. There is one small suggestion here is moving out the `GlobalV::ofs_running` from this function body and add one param like `ofstream& ofs`, then inside you use `ofs` instead of `GlobalV::ofs_running`.